eng/common/scripts/Add-Issue-Comment.ps1

```diff
@@ -23,12 +23,7 @@ param(
 )
 
 . "${PSScriptRoot}\logging.ps1"
-
-$headers = @{
-  Authorization = "bearer $AuthToken"
-}
-
-$apiUrl = "https://api.github.com/repos/$RepoOwner/$RepoName/issues/$IssueNumber/comments"
+. "${PSScriptRoot}\Invoke-GitHub-API.ps1" -AuthToken $AuthToken
```

Comment:
We should move this into common.ps1 and import common.ps1 here instead of this and logging independently.